### PR TITLE
fix: do not update snapshot or fix eslint when running in CI

### PR DESCRIPTION
**Pull request by agdimech**
Fix issue with snapshots being cached when they shouldn't be.

### CODE REVIEW

**Review by JeremyJonas — DISMISSED** [review]

**Review by cogwirrel — DISMISSED** [review]

File: private/pdk-project.ts [hunk 1]
```diff
@@ -145,6 +149,33 @@ export class PDKProject extends JsiiProject {
...
     this.pdkRelease = new PDKRelease(this);
   }
+
+  /**
+   * Provides the ability to override project specific NX Project Targets.
+   *
+   * @return Nx ProjectTargets specific to this package.
+   */
+  public getNxProjectTargets(): ProjectTargets {
+    const relativeDir = `${this.outdir.split(this.root.outdir)[1]}`;
```
Comment (JeremyJonas) [inline private/pdk-project.ts]: Is there reason not to use nx built-in var`{projectRoot}`? - https://nx.dev/more-concepts/customizing-inputs

File: private/projects/aws-prototyping-sdk-project.ts [hunk 2]
```diff
@@ -146,4 +148,25 @@ export class AwsPrototypingSdkProject extends PDKProject {
 
     super.synth();
   }
+
+  /**
+   * @inheritDoc
+   */
+  getNxProjectTargets(): ProjectTargets {
+    const defaultTargets = super.getNxProjectTargets();
+    const relativeDir = `${this.outdir.split(this.root.outdir)[1]}`;
```
Comment (JeremyJonas) [inline private/projects/aws-prototyping-sdk-project.ts]: Should we expose this as a getter on PDKProject? Seems useful

**Review by JeremyJonas — DISMISSED** [review]
Stellar fix and improvements, thank you!

File: private/pdk-project.ts (hunk 1 above)
Comment (agdimech) [reply private/pdk-project.ts]: DIdn't know this existed! Will refactor :)

File: private/projects/aws-prototyping-sdk-project.ts (hunk 2 above)
Comment (agdimech) [reply private/projects/aws-prototyping-sdk-project.ts]: Ill keep as is for now as it's an internal construct. If needed we can make public later.

**Review by JeremyJonas — DISMISSED** [review]

**Review by JeremyJonas — APPROVED** [review]

**Review by JeremyJonas — APPROVED** [review]

**Pull request merged**